ibind/support/logs.py

```diff
@@ -1,4 +1,5 @@
-import datetime
+from datetime import datetime, timezone
```

Comment:
Rather than importing from the `datetime` package, can you stick to the namespace usage already present in this project? Therefore, use `datetime.datetime.now`, and `datetime.timezone.utc`, instead of `datetime.now` and `timezone.utc`. Other than that, looks good! Thanks 👍